css/styles.layout.css

```
@@ -718,24 +726,598 @@
         display: grid;
         gap: 12px;
-        min-height: 0;
-        overflow: auto;
-        overscroll-behavior: contain;
-        scrollbar-gutter: stable;
-        padding-left: 2px;
-        padding-right: 6px;
+        animation: syncSlideDown 0.2s ease-out;
+      }
+
+      @keyframes syncSlideDown {
```

Stylelint (17.7.0)

[error] Expected keyframe name "syncSlideDown" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)